packages/flutter_test/lib/src/matchers.dart

```diff
@@ -323,11 +323,11 @@ Matcher coversSameAreaAs(Path expectedPath, { @required Rect areaToCompare, int
 ///    verify that two different code paths create identical images.
```

Comment:
Can you update the doc comment to explain what the version is for?

Reply:
Done.